<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSecurityWindow</class>
 <widget class="QMainWindow" name="ConfigSecurityWindow">
  <property name="geometry">
   <rect>
       <x>0</x>
       <y>0</y>
       <width>324</width>
       <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Configurações de Seguraça&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_Vin">
          <property name="text">
           <string>Fator VcMáx</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_fvcmax">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_Vo">
          <property name="text">
           <string>Fator VdMáx</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
             <property name="sizeHint" stdset="0">
                 <size>
                     <width>40</width>
                     <height>20</height>
                 </size>
             </property>
         </spacer>
        </item>
           <item>
               <widget class="QLineEdit" name="input_fvdmax">
                   <property name="maximumSize">
                       <size>
                           <width>60</width>
                           <height>16777215</height>
                       </size>
                   </property>
               </widget>
           </item>
       </layout>
      </item>
         <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                     <widget class="QLabel" name="label_Vin_2">
                         <property name="text">
                             <string>Fator VcMáx</string>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <spacer name="horizontalSpacer_13">
                         <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                             <size>
                                 <width>40</width>
                                 <height>20</height>
                             </size>
                         </property>
                     </spacer>
                 </item>
                 <item>
                     <widget class="QLineEdit" name="input_fvsmax">
                         <property name="maximumSize">
                             <size>
                                 <width>60</width>
                                 <height>16777215</height>
                             </size>
                         </property>
                     </widget>
                 </item>
             </layout>
         </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_Vo_2">
          <property name="text">
           <string>Fator Icef Máx</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_ficmax">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_DeltaIin">
          <property name="text">
           <string>Fator Idef Máx</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
             <property name="sizeHint" stdset="0">
                 <size>
                     <width>40</width>
                     <height>20</height>
                 </size>
             </property>
         </spacer>
        </item>
           <item>
               <widget class="QLineEdit" name="input_fidmax">
                   <property name="maximumSize">
                       <size>
                           <width>60</width>
                           <height>16777215</height>
                       </size>
                   </property>
               </widget>
           </item>
       </layout>
      </item>
         <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                     <widget class="QLabel" name="label_Vin_3">
                         <property name="text">
                             <string>Fator IsMáx</string>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <spacer name="horizontalSpacer_14">
                         <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                             <size>
                                 <width>40</width>
                                 <height>20</height>
                             </size>
                         </property>
                     </spacer>
                 </item>
                 <item>
                     <widget class="QLineEdit" name="input_ismax">
                         <property name="maximumSize">
                             <size>
                                 <width>60</width>
                                 <height>16777215</height>
                             </size>
                         </property>
                     </widget>
                 </item>
             </layout>
         </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_DeltaVo">
          <property name="text">
           <string>Fu transformador</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_futrafo">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_JMax">
          <property name="text">
           <string>Fu indutor de entrada</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="input_fuLi">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_DeltaVo_2">
          <property name="text">
           <string>Fu indutor auxiliar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
           <item>
               <widget class="QLineEdit" name="input_fuLk">
                   <property name="maximumSize">
                       <size>
                           <width>60</width>
                           <height>16777215</height>
                       </size>
                   </property>
               </widget>
           </item>
       </layout>
      </item>
         <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                     <widget class="QLabel" name="label_DeltaVo_3">
                         <property name="text">
                             <string>Jmax</string>
                         </property>
                     </widget>
                 </item>
                 <item>
                     <spacer name="horizontalSpacer_12">
                         <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                             <size>
                                 <width>40</width>
                                 <height>20</height>
                             </size>
                         </property>
                     </spacer>
                 </item>
                 <item>
                     <widget class="QLineEdit" name="input_Jmax">
                         <property name="maximumSize">
                             <size>
                                 <width>60</width>
                                 <height>16777215</height>
                             </size>
                         </property>
                     </widget>
                 </item>
             </layout>
         </item>
         <item>
             <spacer name="verticalSpacer_2">
                 <property name="orientation">
                     <enum>Qt::Vertical</enum>
                 </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="save_configurations_button">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
